src/edu/psu/sweng500/team8/gui/BoardGUI.java

```diff
@@ -123,10 +124,20 @@ private void cellGainedFocus(FocusEvent focusEvent) {
 
 	}
 
+	/**
+	 * 
+	 * cellLostFocus is no longer needed as we manually handle the SelectedCell

```

Comment:
Another practice differences. When merging multiple branches without being 100% sure the method is being updated by other or not, it's better to first deprecate the method with a comment and then delete it later when it's merged and stable. Since we know that no one else is updating, I will just go ahead and delete.